search/utils.hpp

```diff
@@ -86,9 +86,6 @@ bool FillCategories(QuerySliceOnRawStrings<T> const & slice, Locales const & loc
                     CategoriesHolder const & catHolder, std::vector<uint32_t> & types)
```

Comment:
```
// Returns |true| and fills |types| if request specified by |slice| is categorial
// in any of the |locales| and |false| otherwise.
// We expect that categorial requests should mostly arise from clicking on a category 
// button in the UI but also allow typing the category name without errors manually
// and not adding a space.
```

Reply:
спасибо! исправила